.github/actions/release/action.yml

```diff
@@ -0,0 +1,47 @@
+name: GitHub release and Publish to NPM
+description: Create a GitHub release and publish to NPM
+
+inputs:
+  github-token:
+    required: true
+    description: 'The GitHub token to use for authentication'
+  release-tag:
+    required: true
+    description: 'The tag to use for the release'
+  draft:
+    required: false
+    description: 'Whether to create a draft (unpublished) release'
+    default: 'true'
+  prerelease:
+    required: false
+    description: 'Whether to create a prerelease release'
+    default: 'true'
+
+runs:
+  using: 'composite'
+  steps:
+    # - name: Create GitHub Release
+    #   uses: DevCycleHQ/release-action/gh-release@main
+    #   with:
+    #     github-token: ${{ inputs.github-token }}
+    #     draft: ${{ inputs.draft }}
+    #     prerelease: ${{ inputs.prerelease }}
+    #     tag: # may need tag here
```

Comment:
We're going to be making some breaking changes to DevCycleHQ/release-action pretty soon, so I'd say it's not worth keeping a commented version around.

Reply:
ohhh, I see, thanks for the update!